Snakefile

```diff
@@ -166,7 +171,9 @@ if not config["custom_data"]["gas_network"]:
             custom_gas_network=config["custom_data"]["gas_network"],
         input:
             regions_onshore=pypsaearth(
-                "resources/bus_regions/regions_onshore_elec_s{simpl}_{clusters}.geojson"
+                "resources/"
+                + RDIR_PE
```

Comment:
Just for understanding: This means, that the run name in PyPSA-Earth-Sec will also influence the resource folder in the PyPSA-Earth submodule?

Reply:
Currently yes; I added a comment in the previous comment